react-app/src/components/form/DateFormField.tsx

```diff
@@ -41,6 +41,7 @@ const DateFormField = (props: DateFieldFormProps) => {
                   InputLabelProps: {
                     style: { fontSize: '14px' }, // Font size for label
                   },
+                  required: required ? true : undefined,
```

Comment:
This works, but you could also just do this:
```suggestion
                  required: required,
```

Or even this:
```suggestion
                  required,
```